doc/user/content/releases/v0.50.md

```diff
@@ -1,13 +1,29 @@
 ---
 title: "Materialize v0.50"
 date: 2023-04-12
-released: false
+released: true
+patch: 3
 ---
 
-{{< warning >}}
-This version of Materialize is not yet released.
-{{< /warning >}}
+## v0.50.3
```

Comment:
Suggestion: it feels like we should just update the header of the page to whatever patch version is the current one, since it's rare that we flesh out release notes for patch versions, and these are otherwise invisible to the user. This can be confusing e.g. when you type `SELECT version();` and it shows up as `v0.50.3`, though the release notes mention `v0.50.0`. 

If we go this route, we'll need to patch the past release notes for consistency.

Reply:
I think this is a good idea. I'm going to keep this with the rest of the release notes convention for now though and work on an update for previous release notes when it's not a release day.

Reply:
Sounds good! Also worth back-propagating the section breakdown (e.g. Sources and sinks, SQL, Bug fixes and other improvements) then.